jetbrains-core-gui/tst/software/aws/toolkits/jetbrains/ui/wizard/SamInitProjectBuilderIntelliJTest.kt

```diff
@@ -101,26 +101,26 @@ class SamInitProjectBuilderIntelliJTest(private val testParameters: TestParamete
             TestParameters(
                 runtime = "java8",
                 templateName = "AWS SAM Hello World (Maven)",
-                sdk = "1.8",
+                sdkRegex = Regex(".*java.*"),
                 libraries = setOf("Maven: com.amazonaws:aws-lambda-java-core:"),
                 runConfigNames = setOf("[Local] HelloWorldFunction")
             ),
             TestParameters(
                 runtime = "java8",
                 templateName = "AWS SAM Hello World (Gradle)",
-                sdk = "1.8",
+                sdkRegex = Regex(".*java.*"),
                 runConfigNames = setOf("[Local] HelloWorldFunction")
             ),
             TestParameters(
                 runtime = "python3.6",
                 templateName = "AWS SAM Hello World",
-                sdk = "Python",
+                sdkRegex = Regex("Python .*"),
```

Comment:
is python guaranteed to have a space after "Python"?

Reply:
Ya, they always do 'Python 3.7.4'

Reply:
Went ahead and removed it